ush/ufoeval/run_ufo_hofx_test.sh

```diff
@@ -76,10 +118,14 @@ config:
 EOF
 $GDASApp/ush/genYAML --config $workdir/temp.yaml
```

Comment:
While we are modifying this script, what about a check for exit code after each key step so that it quits and doesn't keep going? or is that overkill?

Reply:
Not overkill.  Very useful not to have EVA just ploughing ahead.  

Done